test/core/bad_connection/close_fd_test.cc

```diff
@@ -328,7 +328,6 @@ static void _test_close_before_server_recv(fd_type fdtype) {
    */
   if (event.type == GRPC_QUEUE_TIMEOUT) {
     GPR_ASSERT(event.success == 0);
-    GPR_ASSERT(event.tag == nullptr);
```

Comment:
Please also change the comment of `tag` (https://github.com/grpc/grpc/blob/a3062ca04afef15327512d343cbd1768326b7500/include/grpc/impl/codegen/grpc_types.h#L497) to state explicitly that other types can't assume `tag` initialized. Otherwise, readers may think other types' tag is null.

Reply:
Done.